docs/configure.md

```diff
@@ -209,6 +209,59 @@ environments:
       #auth_mode: token 
 
       #token: enter-your-token
+
+      #####################################################
+      # Okta Authentication                              #
+      #####################################################
+     
+      # Indicate the environment is using oauth
+      #auth_mode: oauth 
+
+      #====================================#
+      # Oauth Option 1:                    #
+      # (Unoffical)TNCO Client credentials #
```

Comment:
This says TNCO client credentials but it's actually Okta from my understanding? Also is the `(Unofficial)` part true?